lib/services/users.pm

```diff
@@ -193,4 +193,19 @@ sub full_users_check {
     }
 }
 
+# cleanup for exceptions drung before and after migration
+sub users_cleanup {
+    my (%hash) = @_;
+    my $stage = $hash{stage};
+    select_console "root-console";
+    if ($stage eq 'before') {
+        script_run('pkill -KILL -u test');
+        script_run('userdel -r test');
+        reset_consoles;
```

Comment:
move this to the last line since each branch need run reset_consoles.

Reply:
OK